source/browseMode.py

```diff
@@ -440,9 +442,53 @@ def _iterNodesByType(self,itemType,direction="next",pos=None):
 	def _iterNotLinkBlock(self, direction="next", pos=None):
 		raise NotImplementedError
 
+	def _iterSimilarParagraph(
+			self,
+			kind: str,
+			paragraphFunction: Callable[[textInfos.TextInfo], Optional[Any]],
+			desiredValue: Optional[Any] = None,
+			direction: str = "next",
+			pos: textInfos.TextInfo = None,
+	) -> Generator[TextInfoQuickNavItem, None, None]:
+		info = pos.copy()
+		info.collapse()
+		info.expand(textInfos.UNIT_PARAGRAPH)
+		if desiredValue is None:
+			desiredValue = paragraphFunction(info)
+		while True:
+			# move by one paragraph in the desired direction
+			info.collapse(end=direction == 'next')
+			if direction == 'previous':
+				if info.move(textInfos.UNIT_CHARACTER, -1) == 0:
+					return
+			info.expand(textInfos.UNIT_PARAGRAPH)
+			if info.isCollapsed:
+				return
+			value = paragraphFunction(info)
+			if value == desiredValue:
+				yield TextInfoQuickNavItem(kind, self, info.copy())
+
+
 	def _quickNavScript(self,gesture, itemType, direction, errorMessage, readUnit):
 		if itemType=="notLinkBlock":
 			iterFactory=self._iterNotLinkBlock
+		elif itemType == "textParagraph":
+			punctuationMarksRegex = re.compile(
+				config.conf["virtualBuffers"]["textParagraphRegex"],
+				re.UNICODE,

```

Comment:
`re.UNICODE` is the default in Python 3, so it can be omitted. The same goes for the second usage further down.